feat: Add "Speeding up migrations" section to "State persistence" (#1365)
apify/apify-docs: sources/platform/actors/development/builds_and_runs/state_persistence.md

</Tabs>

For improved Actor performance consider [caching repeated page data](/academy/expert-scraping-with-apify/saving-useful-stats).

## Speeding up migrations

> Once your Actor receives the `migrating` event, in under a minute, the Apify platform will shut it down and restart it on a new server.
> If you want to speed this process up, once you have persisted the Actor state,
> you can reboot the Actor manually in the `migrating` event handler using the `Actor.reboot()` method
> available in the [Apify SDK for JavaScript](/sdk/js/reference/class/Actor#reboot) or [Apify SDK for Python](/sdk/python/reference/class/Actor#reboot).

Reviewer: ```suggestion
Once your Actor receives the `migrating` event, the Apify platform will shut it down and restart it on a new server within one minute.
To speed this process up, once you have persisted the Actor state,
you can manually reboot the Actor in the `migrating` event handler using the `Actor.reboot()` method
available in the [Apify SDK for JavaScript](/sdk/js/reference/class/Actor#reboot) or [Apify SDK for Python](/sdk/python/reference/class/Actor#reboot).
```
